dist/quick-release-notes/quick-release-notes.go

```diff
@@ -0,0 +1,431 @@
+			spawnedThread = true
+		}
+	}
+	if spawnedThread {
+		handleChannels(createReleaseChannel, createReleaseWaitGroup)
+	} else {
+		close(createReleaseChannel)
+	}
+}
+
+func updateReleasesIfChanged(releaseMetaData releaseMetaData, authToken string, repoPath string) {
+	updateReleaseWaitGroup := &sync.WaitGroup{}
+	updateChannel := make(chan bool, len(releaseMetaData.ReleaseVersionTagMap))
+
+	repoOwner, repoName := splitAndEncodeURLPath(repoPath)
+	releases := getReleases(authToken, repoPath)
+
+	spawnedThread := false
+	for _, release := range releases {
+		if release.Body != releaseMetaData.ReleaseVersionNoteMap[release.Name] {
+			requestURL := fmt.Sprintf("https://api.github.com/repos/%v/%v/releases/%v", repoOwner, repoName, release.ID)
+			patchRequest := releaseRequest{
+				authToken:     authToken,
+				repoPath:      repoPath,
+				requestType:   "PATCH",
+				tagForRelease: release.TagName,
+				versionTitle:  release.Name,
+				releaseNote:   releaseMetaData.ReleaseVersionNoteMap[release.Name],
+			}
+			updateReleaseWaitGroup.Add(1)
+			go updateOrCreateGitHubRelease(patchRequest, requestURL, updateChannel, updateReleaseWaitGroup)
+			spawnedThread = true
+		}
+	}
+	if spawnedThread {
+		handleChannels(updateChannel, updateReleaseWaitGroup)
+	} else {
+		close(updateChannel)
+	}
+}
+
+func validateTagsHaveReleases(releaseTagValidationInput releaseTagValidationInput) releaseMetaData {
+	releaseMetaData := releaseMetaData{
+		ReleaseVersionNoteMap: make(map[string]string),
+		ReleaseVersionTagMap:  make(map[string]string),
+		TagReleaseVersionMap:  make(map[string]string),
+	}
+	for i, releaseVersion := range releaseTagValidationInput.changeLog.versions[:] {
+		tagVersionForMap := ""
+		releaseVersionHasTag := false
+		for _, tagVersion := range releaseTagValidationInput.tags[:] {
+			if releaseVersion.number == tagVersion.Name {
+				releaseVersionHasTag = true
+				tagVersionForMap = tagVersion.Name
+			}
+		}
+		if releaseVersionHasTag == false {
+			fmt.Println(redColor+"WARNING: release", releaseVersion, "missing tag"+resetColor)
+		}
+		if releaseVersionHasTag {
+			releaseMetaData.ReleaseVersionTagMap[releaseVersion.number] = tagVersionForMap
+			releaseMetaData.ReleaseVersionNoteMap[releaseVersion.number] = releaseTagValidationInput.releaseNotes[i]
+		}
+	}
+	for _, tagVersion := range releaseTagValidationInput.tags[:] {
+		releaseVersionForMap := ""
+		tagHasVersionNumber := false
+		for _, releaseVersion := range releaseTagValidationInput.changeLog.versions[:] {
+			if tagVersion.Name == releaseVersion.number {
+				tagHasVersionNumber = true
+				releaseVersionForMap = releaseVersion.number
+			}
+		}
+		if tagHasVersionNumber == false {
+			fmt.Println(redColor+"WARNING: tag", tagVersion.Name, "missing changelog entry"+resetColor)
+		}
+		if tagHasVersionNumber {
+			releaseMetaData.TagReleaseVersionMap[tagVersion.Name] = releaseVersionForMap
```

Comment:
Same comment as with `tagVersionForMap` above, but with `releaseVersionForMap` this time.